packages/edit-site/src/components/sidebar-edit-mode/page-panels/page-summary.js

```diff
@@ -18,7 +22,7 @@ export default function PageSummary( {
 	postType,
 } ) {
 	return (
-		<VStack>
+		<VStack spacing={ 0 }>
```

Comment:
This small change is independent but it just removes the spacing between the different controls to match the post editor.